travis/build_info.py

```diff
@@ -0,0 +1,207 @@
+"""
+Gather Travis usage info in real-time using Travis REST APIs
+
+This only applies to travis instances that do not require
+authorization (e.g., travis-ci.org). Auth is a #TODO
+
+"""
+
+import argparse
+import logging
+import requests
+import sys
+
+
+logger = logging.getLogger(__name__)
+
+BASE_URL = 'https://api.travis-ci.org/'
+
+
+def get_repos(org):
+    """
+    Returns list of active repos in a given org.
+    """
+
+    repo_list = []
+    # Note: One call is needed if org has <= 100 active repos.
+    # If org has > 100, this code will need to be modified
+    #  to include 'offset' requests
```

Comment:
We are currently at 67 that are configured for Travis usage. However, the number that actually runs builds is below that (maybe 1 or 2 dozen). From what I can tell, the Travis API lists according to the recent usage. But obviously an off-hand observation is not a solution. I'll put in a TE ticket to address this.